modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/fieldEditableReducer.es.js

```diff
@@ -646,11 +646,19 @@ export default function fieldEditableReducer(state, action, config) {
 				fieldHovered: action.payload,
 			};
 		case EVENT_TYPES.DND.MOVE: {
-			state.focusedField = FormSupport.findFieldByFieldName(
+			if (!state.focusedField.fieldName) {
+				return state;
+			}
+
+			const updatedFocusedField = FormSupport.findFieldByFieldName(
 				state.pages,
 				state.focusedField.fieldName
 			);
 
+			if (updatedFocusedField) {
+				state.focusedField = updatedFocusedField;
+			}
+
```

Comment:
Hi @larissacribeiro,
sorry for in some previous PR I had approved setting something into state.focusedField. This is not right, although it seems to work. We only use state.focusedField to consult its value, not to set a value into it. So lets do some changes by extracting the focused field and pages from the state. Please, could you test if this code works as expected?
```
case EVENT_TYPES.DND.MOVE: {
			const {focusedField, pages} = state;

			if (!focusedField) {
				return state;
			}

			const updatedFocusedField = FormSupport.findFieldByFieldName(
				pages,
				focusedField.fieldName
			);


			return {
			       focusedField: updatedFocusedField
                        };
		}
```

Reply:
Hey, @alinedoleron, I see! 
I just did the test here and the bug was still happening, since the focusedField is always an object, it would not go through the first condition, so I only passed the fieldName attribute to it. Is it ok?

```
		case EVENT_TYPES.DND.MOVE: {
			const {focusedField, pages} = state;

			if(!focusedField.fieldName) {
				return state;
			}

			const updatedFocusedField = FormSupport.findFieldByFieldName(
				pages,
				focusedField.fieldName
			);

			return {
				focusedField: updatedFocusedField
			};
		}
```

Reply:
Hi @larissacribeiro. I got it! You're right!